askbot/models/post.py

```diff
@@ -173,6 +173,8 @@ def get_answers(self, user=None):
         optionally filtered to exclude posts of groups
         to which user does not belong"""
         answers = self.filter(post_type='answer')
+        if user == 'n/a':
```

Comment:
None does exactly the wrong thing (for a private question, zero posts are returned to Thread's update_last_activity_info()). I agree the 'n/a' feels nasty, but I felt it was better than replicating a whole bunch of code copy-and-paste style. I can take another look at this next week, see if I can find something that feels better though.